helix-loader/Cargo.toml

```diff
@@ -15,8 +15,7 @@ serde = { version = "1.0", features = ["derive"] }
 toml = "0.5"
 etcetera = "0.4"
 tree-sitter = "0.20"
-once_cell = "1.12"
-
+once_cell = "1.9"
```

Comment:
Dependency was downgraded?

Reply:
Oops, I think this was a bad rebase conflict resolution. Will fix.